src/debugging.js

```diff
@@ -77,9 +77,11 @@ export function getConfig(debugging) {
 }
 config.getConfig('debugging', ({debugging}) => getConfig(debugging));
 
-export function sessionLoader(storage = window.sessionStorage) {
+export function sessionLoader() {
   let overrides;
   try {
+    if (!window.localStorage || !window.sessionStorage) throw "exception";
```

Comment:
Is this line needed since if they don't exist an exception will be thrown when `storage` is used on line 85?

Also, if we do want to keep this line, this should be `!window.localStorage && !window.sessionStorage`